library/src/main/java/com/digi/xbee/api/DigiMeshDevice.java

```diff
@@ -424,4 +430,35 @@ public void sendExplicitDataAsync(XBee64BitAddress address, int sourceEndpoint,
 			int profileID, byte[] data) throws XBeeException {
 		super.sendExplicitDataAsync(address, sourceEndpoint, destEndpoint, clusterID, profileID, data);
 	}
+	
+	/**
+	 * @deprecated DigiMesh protocol does not have an associated IPv6 address.
+	 */
+	@Override
+	public Inet6Address getIPv6Address() {
+		// DigiMesh protocol does not have IPv6 address.
+		return null;
```

Comment:
It can be changed to throw an exception. It was decided to leave it this way because getDestAddress() method is not applicable to the devices that don't support IPv6 addressing (it is more of a concept kind of decision), whereas, since AbstractXBee device has an ipv6Address attribute, technically we should return something for that attribute. As said before, it can be changed.